webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/fields/TokenLifeTime.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  FormGroup,
+  TextInput,
+  InputGroup,
+  InputGroupText,
+  Checkbox,
+  Popover,
+} from '@patternfly/react-core';
+
+import { HelpIcon } from '@patternfly/react-icons';
+import { sprintf, translate as __ } from '../../../../../common/I18n';
+
+const TokenLifeTime = ({ value, onChange, handleInvalidField, isLoading }) => {
+  const minValue = 1;
+  const maxValue = 999999;
+
+  const isValid = v => {
+    if (v === 'unlimited') {
+      return true;
+    }
+
+    return v >= minValue && v <= maxValue;
+  };
+
+  const setValue = v => {
+    handleInvalidField('Token Life Time', isValid(v));
+    onChange(v);
+  };
+
+  return (
+    <FormGroup
+      label={__('Token Life Time (hours)')}
+      validated={isValid(value) ? 'default' : 'error'}
+      helperTextInvalid={sprintf(
+        'Token life time value must be between %s and %s hours.',
+        minValue,
+        maxValue
+      )}
+      isRequired
+      fieldId="registration_token_life_time"
+      labelIcon={
+        <Popover
+          bodyContent={
+            <div>{__('Expiration of the authorization token.')}</div>
+          }
+        >
+          <button
+            className="pf-c-form__group-label-help"
+            onClick={e => e.preventDefault()}
+          >
+            <HelpIcon noVerticalAlign />
+          </button>
+        </Popover>
+      }
+    >
+      <InputGroup>
+        <TextInput
+          value={value}
+          type="number"
+          min={minValue}
+          max={maxValue}
+          validated={isValid(value) ? 'default' : 'error'}
+          isDisabled={isLoading || value === 'unlimited'}
+          id="reg_token_life_time_input"
+          onChange={v => setValue(v)}
+        />
+        <InputGroupText>hours</InputGroupText>
+        <InputGroupText>
+          <Checkbox
```

Comment:
This one also:
![Screenshot (86)](https://user-images.githubusercontent.com/26363699/113706916-e051c880-96e7-11eb-8409-426c77a0e4ee.png)